src/ReverseProxy/Routing/DirectForwardingIEndpointRouteBuilderExtensions.cs

```diff
@@ -22,6 +24,46 @@ public static IEndpointConventionBuilder MapForwarder(this IEndpointRouteBuilder
         return endpoints.MapForwarder(pattern, destinationPrefix, ForwarderRequestConfig.Empty);
     }
 
+    /// <summary>
+    /// Adds direct forwarding of HTTP requests that match the specified pattern to a specific destination and target path applying route values from the pattern using default configuration for the outgoing request, and default HTTP client.
+    /// </summary>
+    public static IEndpointConventionBuilder MapForwarder(this IEndpointRouteBuilder endpoints, string pattern, string destinationPrefix, string targetPath)
+    {
+        return endpoints.MapForwarder(pattern, destinationPrefix, ForwarderRequestConfig.Empty, targetPath);
+    }
+
+    /// <summary>
+    /// Adds direct forwarding of HTTP requests that match the specified pattern to a specific destination and target path applying route values from the pattern using customized configuration for the outgoing request, and default HTTP client.
+    /// </summary>
+    public static IEndpointConventionBuilder MapForwarder(this IEndpointRouteBuilder endpoints, string pattern, string destinationPrefix, ForwarderRequestConfig requestConfig, string targetPath)
+    {
+        return endpoints.MapForwarder(pattern, destinationPrefix, requestConfig, b => b.AddPathRouteValues(targetPath));
+    }
+
+    /// <summary>
+    /// Adds direct forwarding of HTTP requests that match the specified pattern to a specific destination using default configuration for the outgoing request, customized transforms, and default HTTP client.
+    /// </summary>
+    public static IEndpointConventionBuilder MapForwarder(this IEndpointRouteBuilder endpoints, string pattern, string destinationPrefix, Action<TransformBuilderContext> configureTransform)
+    {
+        var transformBuilder = endpoints.ServiceProvider.GetRequiredService<ITransformBuilder>();
+
+        var transformer = transformBuilder.Create(configureTransform);
+
+        return endpoints.MapForwarder(pattern, destinationPrefix, ForwarderRequestConfig.Empty, transformer);
```

Comment:
```suggestion
        return endpoints.MapForwarder(pattern, destinationPrefix, ForwarderRequestConfig.Empty, configureTransform);
```
Nit: We can de-dup these

Reply:
Done